controllers/apply/awards-for-all/fields.js

```diff
@@ -2067,29 +2067,5 @@ module.exports = function fieldsFor({ locale, data = {} }) {
         }
     };
 
-    const schema = Joi.object(
```

Comment:
This is now computed by the form model rather than needing to do this per form and pass properties all the way down.